parquet-column/src/main/java/org/apache/parquet/column/statistics/geospatial/BoundingBox.java

```diff
@@ -218,16 +233,27 @@ public void merge(BoundingBox other) {
       return;
     }
 
-    this.xMin = Math.min(this.xMin, other.xMin);
-    this.xMax = Math.max(this.xMax, other.xMax);
+    // We don't yet support merging wraparound bounds.
+    // Rather than throw, we mark the X bounds as invalid.
```

Comment:
```suggestion
    // We don't yet support merging wraparound bounds.
```

If you accept the below suggestion, this line should also be removed.